src/main/java/it/pagopa/ecommerce/commons/documents/v1/TransactionActivatedData.java

```diff
@@ -27,5 +28,6 @@ public class TransactionActivatedData {
     private Transaction.ClientId clientId;
     @Nullable
     private String idCart;
+    private BigInteger paymentTokenValiditySeconds;
```

Comment:
```2^32 / 60 / 60 ~= 1 * 10^6 hours```

Can we just use `int` here?

Reply:
Fixed with https://github.com/pagopa/pagopa-ecommerce-commons/pull/84/commits/f77d6903716f4a2d22df50d889e6a629742fef33